library/packages/src/lib/y2packager/licenses_handlers.rb

```diff
@@ -19,13 +20,16 @@ module Y2Packager
   # classes defined in this module are able to interact with these sources in
   # order to find out, for instance, whether a license must be accepted.
   module LicensesHandlers
-    # Return the licenses proper fetcher for a given source
+    # Return the proper license handler based on given fetcher
+    #
+    # @param fetcher      [LicensesFetchers::Base] Fetcher used as source to fetch license
+    # @param product_name [String]                 Product's name
     #
-    # @param source       [:libzypp,nil] Source to fetch license from (only :libzypp is supported)
-    # @param product_name [String]       Product's name
     # @return [Object]
```

Comment:
Is it really generic object or Fetcher child?

Reply:
A LicensesHandlers child, which at the end is an Object. I'll change the documentation to `LicensesHandlers::Base` :)